tests/supported_runtimes_test.rs

```diff
@@ -46,6 +111,7 @@ echo Hello, $name!
         .success();
 }
 
+#[cfg(not(windows))]
 #[test]
 fn bash() {
     let (_temp, maskfile_path) = common::maskfile(
```

Comment:
Huge thank you for all these Windows test equivalents! 😄

Reply:
Sure, wanted this to be as complete as possible.